backend/icpy/services/chat_service.py

```
@@ -567,14 +697,52 @@ async def _process_with_custom_agent(self, user_message: ChatMessage, agent_type
+                            rel = att.get('relative_path') or att.get('rel_path') or att.get('path')
+                            data_url = None
+                            if isinstance(rel, str) and rel:
+                                try:
+                                    abs_path = (media.base_dir / rel).resolve()
+                                    abs_path.relative_to(media.base_dir)
+                                    if abs_path.exists() and abs_path.is_file():
+                                        import base64
+                                        with open(abs_path, 'rb') as f:
+                                            b64 = base64.b64encode(f.read()).decode('ascii')
+                                        data_url = f"data:{mime};base64,{b64}"
+                                except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)